arangod/Aql/TraversalNode.h

```diff
@@ -245,6 +246,9 @@ class TraversalNode : public virtual GraphNode {
   void traversalCloneHelper(ExecutionPlan& plan, TraversalNode& c,
                             bool withProperties) const;
 
+  /// @brief variables that got optimized out
+  VarIdSet _optimizedOutVariables;
```

Comment:
Why not use `VarSet` for this?

Reply:
We do not need the full variable. We just need the register id (uint32_t). Also we need to serialize and deserialize this. Should be smaller then using a full "Variable".

Reply:
That's true, but adds more variation (all other places uses `VarSets`).
Anyway, I can live with how it is right now.